src/convertToObject.js

```diff
@@ -12,7 +12,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  for (const style of sourceString.split(';')) {
+    const [key, value] = style.split(':');
+
+    if (key.trim() !== '' && value.trim() !== '') {
+      styles[key.trim()] = value.trim();
```

Comment:
you are using the same command twice (`key.trim()`). At this point you can create a variable for that. the same goes for `value.trim()`.